canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.scss

```diff
@@ -39,6 +39,11 @@ $row-left-padding: $spacing-02;
 	height: $spacing-07;
 }
 
+.single-table {
```

Comment:
This class isn't used anywhere other than this file

Reply:
I removed it on accident when merging. I used it to fix the following bug. Screen Recording 2024-10-17 at 9.49.44 AM.mov

Reply:
Rename to `single-row-selection-table`